src/bundle/Controller/SearchController.php

```diff
@@ -128,6 +128,7 @@ public function searchAction(Request $request): Response
                 $contentTypes = $data->getContentTypes();
                 $lastModified = $data->getLastModified();
                 $created = $data->getCreated();
+                $filtersExpanded = $data->isFiltered();
```

Comment:
Since you are using this value only once I don't think you need to declare the variable. `$data->isFiltered()` is also self explanatory.